projects/plugins/crm/includes/ZeroBSCRM.FormatHelpers.php

```diff
@@ -850,6 +850,41 @@ function zeroBSCRM_html_quoteStatusLabel($quote=array()){
 
 	}
 
+/**
+ * Return an updated HTML string, replacing date placeholders with correct date strings based on site settings.
+ * @param  string $value                 The value of the date variable to be replaced.
+ * @param  string $key                   The key of the date variable to be replaced.
+ * @param  string $working_html          The HTML string to be updated.
+ * @param  string $placeholder_str_start The string to add to the beginning of the placeholder string (eg. ##QUOTE-).
+ *
+ * @return string                The updated HTML string.
+ */
+function jpcrm_process_date_variables( $value, $key, $working_html, $placeholder_str_start = '##' ) {
+
+	$base_date_key       = $key;
+	$string_to_base_date = jpcrm_uts_to_date_str( strtotime( $value ), get_option( 'date_format' ) );
```

Comment:
As best I understand it, this is the intent (`strtotime` comments aside):
* `$string_to_base_date` → date in WP format
* `$string_to_datetime` → datetime in hybrid hard-coded date + WP time format...why not `jpcrm_uts_to_datetime_str()`?
* `$string_to_date` → hard-coded date

Three thoughts:
1. For the datetime one, does this make sense to include? It's always a date being passed, not a datetime, so the time part would be some arbitrary time depending on what functions we used.
2. For the third one, perhaps we should hard-code to the ISO standard instead of an arbitrary "common (in English)" format?
3. We're basically creating undocumented placeholders here (e.g. they won't show in `/wp-admin/admin.php?page=zerobscrm-systemstatus&tab=placeholder-map`). Any reason we need all of these? The only one I see documented is `##QUOTE-DATE##`.

Reply:
We chatted in Slack, following up here to say that I'll follow that suggested approach - using the in-built functions, following the style in the similar invoice date PR: https://github.com/Automattic/zero-bs-crm/pull/2239
So the base date is actually the ISO standard (except for Quote Date itself, which I'll leave with just one available placeholder). 
For point 3, to clarify here too we're only checking for strings to search and replace for quote custom fields in this case, which all have three available placeholder options.